src/components/PreviewFrame.jsx

```diff
@@ -39,7 +35,7 @@ class PreviewFrame extends React.Component {
           name={`preview-frame-${nextId++}`}
           ref={this._attachToFrame}
           sandbox={sandboxOptions}
```

Comment:
I could be wrong but it seems that having the sandbox setting on at all prevents us from accessing the camera/audio regardless of if we load the HTML from the same origin.

Reply:
OK! Per Slack conversation let’s merge this anyway and then keep hammering on the camera API.